code/train_LSTM.py

```diff
@@ -37,7 +37,7 @@
 
 flags.DEFINE_integer('minibatch_size', 16, 'Number of Mini batches executed per epoch (batch).')
 
-flags.DEFINE_integer('num_classes', 7, 'Number of classes to train on')
+flags.DEFINE_integer('num_classes', 10, 'Number of classes to train on')
```

Comment:
Why increase this?

Reply:
Don't just resolve this without addressing my comment.

Reply:
Back to 7.